doc/api/cli.md

```diff
@@ -950,16 +950,35 @@ When set to `1`, process warnings are silenced.
 added: v8.0.0
 -->
 
-A space-separated list of command line options. `options...` are interpreted as
-if they had been specified on the command line before the actual command line
-(so they can be overridden). Node.js will exit with an error if an option
-that is not allowed in the environment is used, such as `-p` or a script file.
+A space-separated list of command line options. `options...` are interpreted
+before command line options, so command line options will override or
+compound after anything in `options...`. Node.js will exit with an error if
+an option that is not allowed in the environment is used, such as `-p` or a
+script file.
 
-In case an option value happens to contain a space (for example a path listed in
-`--require`), it must be escaped using double quotes. For example:
+In case an option value happens to contain a space (for example a path listed
+in `--require`), it must be escaped using double quotes. For example:
 
 ```bash
---require "./my path/file.js"
+NODE_OPTIONS='--require "./my path/file.js"'
+```
+
+A singleton flag passed as a command line option will *override* the same flag
```

Comment:
Nit: No need for emphasis? Also, _singleton_ is confusing here, so maybe omit?

```suggestion
A singleton flag passed as a command line option will override the same flag
```

Reply:
Removed the emphasis, but left singleton in -- happy to follow what you think on that, I think that it can work either way.

Reply:
Yeah, I changed my mind and decided `singleton` was fine. If there's a better word, it can always be updated in a subsequent PR.